x/distribution/types/msg.go

```diff
@@ -24,6 +27,11 @@ var (
 	_ sdk.Msg = (*MsgWithdrawValidatorCommission)(nil)
 	_ sdk.Msg = (*MsgUpdateParams)(nil)
 	_ sdk.Msg = (*MsgCommunityPoolSpend)(nil)
+	_ sdk.Msg = (*MsgWithdrawTokenizeShareRecordReward)(nil)
+	_ sdk.Msg = (*MsgWithdrawAllTokenizeShareRecordReward)(nil)
+
+	_ legacytx.LegacyMsg = (*MsgWithdrawTokenizeShareRecordReward)(nil)
```

Comment:
Because simulations in SDK are currently written in a way that expects messages as `legacytx.LegacyMsg` there:
https://github.com/cosmos/cosmos-sdk/blob/10b3db2318f0df1521d00913bfad82c9c0e6d784/types/simulation/types.go#L91-L93

They have already rewritten that part for the next version (whenever it comes).